build.gradle

```diff
@@ -0,0 +1,260 @@
+ *
+ */
+
+plugins {
+    id "com.github.spotbugs" version "4.0.5"
+    id "com.github.johnrengelman.shadow" version "5.2.0"
+    id "de.undercouch.download" version "4.0.4"
+    id "net.researchgate.release" version "2.8.0"
+}
+
+ext.ballerinaLangVersion = project.ballerinaLangVersion
+ext.stdlibMimeVersion = project.stdlibMimeVersion
+ext.stdlibIoVersion = project.stdlibIoVersion
+ext.stdlibHttpVersion = project.stdlibHttpVersion
+ext.stdlibLogVersion = project.stdlibLogVersion
+ext.stdlibStringutilsVersion = project.stdlibStringutilsVersion
+ext.stdlibAuthVersion = project.stdlibAuthVersion
+ext.stdlibCacheVersion = project.stdlibCacheVersion
+ext.stdlibTimeVersion = project.stdlibTimeVersion
+ext.stdlibCryptoVersion = project.stdlibCryptoVersion
+ext.stdlibFileVersion = project.stdlibFileVersion
+ext.stdlibConfigVersion = project.stdlibConfigVersion
+ext.stdlibReflectVersion = project.stdlibReflectVersion
+ext.stdlibRuntimeVersion = project.stdlibRuntimeVersion
+ext.stdlibSystemVersion = project.stdlibSystemVersion
+ext.stdlibTaskVersion = project.stdlibTaskVersion
+
+ext.slf4jVersion = project.slf4jVersion
+ext.ballerinaTomlParserVersion = project.ballerinaTomlParserVersion
+ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
+ext.commonsLang3Version = project.commonsLang3Version
+ext.nettyVersion = project.nettyVersion
+ext.nettyTcnativeVersion = project.nettyTcnativeVersion
+ext.wso2CommonsPoolVersion = project.wso2CommonsPoolVersion
+ext.wso2SnakeYamlVersion = project.wso2SnakeYamlVersion
+ext.bouncycastleVersion = project.bouncycastleVersion
+ext.mimepullVersion = project.mimepullVersion
+ext.testngVersion =project.testngVersion
+
+allprojects {
+    group = project.group
+    version = project.version
+
+    apply plugin: 'jacoco'
+    apply plugin: 'maven-publish'
+
+    repositories {
+        mavenLocal()
+        maven {
+            url = 'https://maven.wso2.org/nexus/content/repositories/releases/'
+        }
+
+        maven {
+            url = 'https://maven.wso2.org/nexus/content/groups/wso2-public/'
+        }
+
+        maven {
+            url = 'https://repo.maven.apache.org/maven2'
+        }
+
+        maven {
+            url = 'https://maven.pkg.github.com/ballerina-platform/ballerina-lang'
+            credentials {
+                username System.getenv("packageUser")
+                password System.getenv("packagePAT")
+            }
+        }
+
+        maven {
+            url 'https://maven.pkg.github.com/ballerina-platform/module-ballerina-mime'
+            credentials {
+                username System.getenv("packageUser")
+                password System.getenv("packagePAT")
+            }
+        }
+        maven {
+            url 'https://maven.pkg.github.com/ballerina-platform/module-ballerina-io'
+            credentials {
+                username System.getenv("packageUser")
+                password System.getenv("packagePAT")
+            }
+        }
+//        maven {
```

Comment:
Any reason to comment them out?

Reply:
With the new service typing design, I have only implemented the basic flow of websocket. Still there are features to be added in the future. For them, we'll need these. For now I have commented them.

Reply:
Ack